test-butler-app/src/main/java/com/linkedin/android/testbutler/ButlerService.java

```diff
@@ -45,6 +45,8 @@
     private PowerManager.WakeLock wakeLock;
     private KeyguardManager.KeyguardLock keyguardLock;
 
+    private GsmDataDisabler gsmDataDisabler;
```

Comment:
Please group this with the other changer/disabler classes (in general, try to match the code style of the existing code)